### DISCUSSION TITLE
Kong Ingress routing issue

### DISCUSSION THREAD

**Discussion by guptadeepali39** [Kong/kong#8246] [Help] [answered]
Hi,

I am facing one issue, I have two applications, two services and two ingresses.
Now for my ingress I have routing rules like:
1. /TESTMOB/*
2. /TEST/*

But while hitting any request the request to TESTMOB is going to TEST.

Any solution for this?  @kikito

**Comment by javierguerragiraldez** [comment, marked as answer]
rules with patterns are not prioritized by length, so you have to add priorities manually.

but, I think in this case, simple paths would work, without the `*`

**Reply by guptadeepali39** [reply]
yes it worked without *.. thanks for helping out
